packages/components/src/tabs/test/index.tsx

```diff
@@ -1172,37 +1172,79 @@ describe( 'Tabs', () => {
 				).not.toBeInTheDocument();
 			} );
 		} );
-
-		describe( 'When `selectOnMove` is `true`', () => {
-			it( 'should automatically select a newly focused tab', async () => {
-				render( <ControlledTabs tabs={ TABS } selectedTabId="beta" /> );
-
-				await press.Tab();
+		describe( 'When `selectedId` is changed by the controlling component', () => {
```

Comment:
Should the tests in this `describe` run against both values of `selectOnMove` (`true`, `false`) ?